.buildbot.sh

```diff
@@ -4,10 +4,18 @@ set -e
 
 export CHERI=$HOME/cheri/output/sdk
 export LD_LIBRARY_PATH=$CHERI/lib
+export CC=$CHERI/bin/clang 
 
-make CC=$CHERI/bin/clang CXX=$CHERI/bin/clang++ LLVM_CONFIG=$CHERI/bin/llvm-config
-make bitcode CC=$CHERI/bin/clang CXX=$CHERI/bin/clang++ 
-make split CC=$CHERI/bin/clang CXX=$CHERI/bin/clang++ LLVM_CONFIG=$CHERI/bin/llvm-config
+make CC=$CC CXX=$CHERI/bin/clang++ LLVM_CONFIG=$CHERI/bin/llvm-config
+make split CC=$CC CXX=$CHERI/bin/clang++ LLVM_CONFIG=$CHERI/bin/llvm-config
 
-mkdir -p out
-./split file.bc -o ./out
+make test-included CC=$CC CXX=$CHERI/bin/clang++ 
+
```

Comment:
Can you force push an update that removes the extra blank line (unless there's a good reason for it, but I think it's just a typo)?

Reply:
It wasn't, I use it to logically separate the different tests. But I will remove it.